src/data-workspace/inputs/generic-input.js

```diff
@@ -56,21 +40,28 @@ export const GenericInput = ({
             return value.trim()
         }
     }
-
     const { input, meta } = useField(fieldname, {
         validate: validatorsByValueType[valueType],
         subscription: { value: true, dirty: true, valid: true },
         format: formatValue,
         formatOnBlur: true,
+        parse: (value) => value ?? '',
     })
-
-    // need to initialize lastSyncedValue
-    useEffect(() => {
-        if (input.value !== undefined) {
-            setLastSyncedValue(input.value)
-        }
-    }, []) // eslint-disable-line react-hooks/exhaustive-deps
-
+    const [lastSyncedValue, setLastSyncedValue] = useState(input.value)
```

Comment:
Thanks @HendrikThePendric. This was actually the first thing I did before looking at using the `useEffect` hook to set the initial value on mount.

I think what I was seeing though was that the `lastSyncValue` continued to update beyond the setting the initial value. Everytime `input.value` changed `lastSyncValue` updated as well. Doing the change on mount with `useEffect` meant that this only update `lastSyncValue` once. Form a user point of view it seemed to work but I was worried about potentially for introducing some form of race condition with future development.

I'll double check this on my machine.

I did have one more idea for how to address this but I need to double check how the `entry-field-input.js` is working with the generic input first. I don't know if we can pass the the initial value for the field as a prop on the generic input at the entry field input level so that it only sets `lastSyncValue` with a nice clean contract.

Reply:
forget what I said in the comment above only updates once as we would like.

 `const [lastSyncedValue, setLastSyncedValue] = useState(input.value)` 

does work perfectly fine and I work be happy for that to go in :-)

Reply:
Actually testing this further I am seeing a problem with it.. let me just do a screen recording

Reply:
No screenshot needed, had a conversation with @HendrikThePendric instead. Pushed the final fix we arrived at :-)